database/frontend/fonts.json

```
@@ -32,5 +32,12 @@
         "url": "https://type-scale.com",
         "category": "frontend",
         "subcategory": "fonts"
+    },
+    {
+        "name": "Fontesk",
+        "description": "Curated web platform font library of the best high quality free fonts to download for commercial and personal use.",
```

[Proofreading] The description was a bit unclear due to its use of the preposition "of". 
```suggestion
        "description": "Curated web platform font library that contains the best high-quality free fonts to download for commercial and personal use.",
```